core/drivers/crypto/stm32/cipher.c

```diff
@@ -119,6 +119,12 @@ static TEE_Result saes_init(union ip_ctx *ip_ctx, bool is_decrypt,
 		TEE_OperationMode mode = TEE_MODE_ILLEGAL_VALUE;
 		TEE_Result res = TEE_ERROR_GENERIC;
 
+		if (!IS_ENABLED(CFG_STM32_SAES_SW_FALLBACK)) {
+			DMSG("STM32 SAES does not support 192bit keys");
```

Comment:
I'm thinking if EMSG is more relevant here or not ? 
It's an error so error message, but maybe EMSG will make it too verbose and we can assume the user will recompile the driver with debug log level to ease debug.

Reply:
Makes sense. I'll switch to `EMSG()`.